wrangler/helpers/plate_helpers.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from typing import Any, Dict, List
+
+from wrangler.constants import PLATE_PURPOSE_ENTITY, PLATE_PURPOSE_STOCK, STUDY_ENTITY, STUDY_HERON
+from wrangler.helpers.general_helpers import get_entity_uuid
+
+logger = logging.getLogger(__name__)
+
+
+def create_plate_body(plate_barcode: str, mlwh_results: List[Dict[str, str]]) -> Dict[str, Any]:
+    wells_content = {}
+    for sample in mlwh_results:
+        well = {
+            "supplier_name": sample["supplier_sample_id"],
+        }
+        wells_content[sample["coordinate"]] = well
```

Comment:
If this is coming form the MLWH, should it be `position` instead of `coordinate`?

```suggestion
        wells_content[sample["position"]] = well
```

Reply:
Thanks, caught this bug while testing locally too. Good spot!